### ISSUE TITLE
arm64 debian build on `apt.packages.shiftkey.dev`

### ISSUE THREAD

**Issue by ckuhtz** [shiftkey/desktop#1004]
### The feature request

Please provide an Debian ARM64 build in addition to AMD64.

### Proposed solution

Create build pipeline for ARM64 Debian packages.

### Additional context

Can't deploy github-desktop on Debian on ARM64.

$ sudo apt search github-desktop
Sorting... Done
Full Text Search... Done
github-desktop/unknown 3.3.8-linux1 amd64
  Simple collaboration from your desktop

**Comment by theofficialgman** [comment]
arm64 debs have been included in the releases for 6 months now (you are welcome)
first release with this was in July 2023 https://github.com/shiftkey/desktop/releases/tag/release-3.2.7-linux1

this issue can be closed as invalid

**Comment by Tarmogos** [comment]
> ### The feature request
> 
> Please provide an Debian ARM64 build in addition to AMD64.
> 
> ### Proposed solution
> 
> Create build pipeline for ARM64 Debian packages.
> 
> ### Additional context
> 
> Can't deploy github-desktop on Debian on ARM64.
> 
> 
> 
> $ sudo apt search github-desktop
> 
> Sorting... Done
> 
> Full Text Search... Done
> 
> github-desktop/unknown 3.3.8-linux1 amd64
> 
>   Simple collaboration from your desktop
> 
> 
> 
> 
> 
>

**Comment by ckuhtz** [comment]
This issue was referencing the install from Debian repo for the arm64 build. Is that fixed?

**Comment by shiftkey** [comment]
No, I need to rewrite how the package feed is laid out on disk to support other architectures (currently it defaults to `any` which is the amd64 channel).
